lib/ansible/modules/windows/win_share.ps1

```diff
@@ -111,26 +111,34 @@ Function NormalizeAccounts
 
 $result = @{
     changed = $false
+    actions = @() # More for debug purposes
```

Comment:
Is it a bad thing, I would have thought if it wasn't there everytime I want to add an action I would need to
* Check if the attribute was already set in `$result`
* Add the action string

Compare to just adding the action string for the current implementation.